mep/books/admin.py

```diff
@@ -44,7 +44,7 @@
     "updated_at",
 ]
 
-WORK_IMPORT_COLUMNS = ["slug", "title", "category"]
```

Comment:
Editing title works but to be consistent for now, let's keep imported columns to nationality, gender, category in this and the previous PR. slug is there as the id/join key.